solr/solr-ref-guide/modules/indexing-guide/pages/indexing-with-tika.adoc

```diff
@@ -77,20 +77,20 @@ For a few examples of how this could be done, see this blog post by Erick Ericks
 
 == Trying out Solr Cell
 
-You can try out the Tika framework using the `schemaless` example included in Solr.
+You can try out the Tika framework using the `techproducts` example included in Solr.
 
-This command will simply start Solr and create a core/collection named "gettingstarted" with the `_default` configset.
+This command will simply start Solr and create a core/collection named `techproducts` with the `techproducts` configset.
 
 [source,bash]
 ----
-bin/solr -e schemaless
+bin/solr -e techproducts
```

Comment:
There is a big difference between these - the schemaless example creates an empty collection, whereas techproduct indexes a bunch of sample docs, which clutters this example. So a suggestion is to either create a new example for this (`bin/solr -e extracting`), or perhaps easier, create the core manually, as suggested below
```suggestion
bin/solr start
bin/solr create -c gettingstarted -d example/techproducts/solr/techproducts/conf
```

Reply:
Addressed in [b744940](https://github.com/apache/solr/pull/822/commits/b744940a0b20b4832ac7f66b05c7464d743f8c2f)